repo2docker/contentproviders/dataverse.json

```diff
@@ -639,16 +639,29 @@
         },
         {
             "description": "The Research Data Repository for the Max Planck Society.",
-            "full_name": "Edmond - Research Data Repository for the Max Planck Society",
+            "full_name": "Edmond - Research Data Repository for the Max Planck Society (old domain)",
             "id": 1795,
             "is_active": true,
             "lat": 48.1366,
             "lng": 11.5771,
             "logo": "https://edmond.mpdl.mpg.de/logos/navbar/logo_for_bright.png",
-            "name": "Edmond",
-            "slug": "",
+            "name": "Edmond (old domain)",
+            "slug": "edmond_mpdl_mpg_de",
             "url": "https://edmond.mpdl.mpg.de",
-            "version": "5.10"
+            "version": "5.14"
+        },
+        {
+            "description": "The Research Data Repository for the Max Planck Society.",
+            "full_name": "Edmond - Research Data Repository for the Max Planck Society",
+            "id": 1796,
```

Comment:
Is this a reference to an external id, or is it managed by repo2docker?

Reply:
Do you mean the "1796"? I just added +1 to the last ID in the list, as everyone did before. So this shouldn't be managed by repo2docker